### PR TITLE
Strings extraction script

### PR DESCRIPTION
**Pull request by eoger**
Related to https://github.com/mozilla/fxa-content-server/issues/2398

### CODE REVIEW

File: scripts/extract_strings.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+# syntax:
+# extract_strings.sh [--mailer-repo ./fxa-auth-mailer] [--content-repo ./fxa-content-server] [--l10n-repo ./fxa-content-server-l10n] train_number
+
```
Comment (jrgm) [inline scripts/extract_strings.sh]: May I suggest `set -o errexit` (a.k.a., `set -e`)

File: scripts/extract_strings.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+(cd $CONTENT_DIR && grunt l10n-extract) &&
+cp -r $CONTENT_DIR/locale/templates/ $L10N_DIR/locale/templates &&
+
+cd $L10N_DIR &&
+git checkout -b merge-train-$TRAIN_NUMBER-strings &&
+./scripts/merge_po.sh ./locale &&
+git add . &&
+git commit -m "merge strings for train $TRAIN_NUMBER" &&
+git push origin merge-train-$TRAIN_NUMBER-strings
```
Comment (vladikoff) [inline scripts/extract_strings.sh]: I suggest not pushing automatically. Instead print the command that the developer should use. 
This allows us to make sure everything went okay during the extraction process before pushing.

In addition I don't use `origin` in some cases, but instead I use `upstream`

File: scripts/extract_strings.sh [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+function usage() {
+    echo "syntax:"
+    echo "extract_strings.sh [--mailer-repo ./fxa-auth-mailer] [--content-repo ./fxa-content-server] [--l10n-repo ./fxa-content-server-l10n] train_number"
+    exit 1
+}
+
+function check_folder() {
+    if [[ ! -d $1 ]]; then
+        echo "Error!"
```
Comment (vladikoff) [inline scripts/extract_strings.sh]: nit: add a more descriptive error here. `Error: failed to find a directory to extract from` or something

**Comment by eoger** [comment]
Updated, thanks guys.

**Comment by zaach** [comment]
Awesome! Tested it out today and it worked well.

**Comment by eoger** [comment]
:+1:

**Comment by shane-tomlinson** [comment]
This makes me happy.

**Pull request merged**